AFL++ expects clang to be installed in $(llvm-config --bindir)/clang and not
in /usr/bin/clang. Fixing this by supplying LLVM_BINDIR as /usr/bin.

--- AFLplusplus-4.00c/utils/aflpp_driver/GNUmakefile
+++ AFLplusplus-4.00c/utils/aflpp_driver/GNUmakefile
@@ -2,7 +2,7 @@
   LLVM_CONFIG=llvm-config
 endif
 
-LLVM_BINDIR = $(shell $(LLVM_CONFIG) --bindir 2>/dev/null)
+LLVM_BINDIR := /usr/bin
 ifneq "" "$(LLVM_BINDIR)"
   LLVM_BINDIR := $(LLVM_BINDIR)/
 endif
--- AFLplusplus-4.00c.bak/GNUmakefile.llvm
+++ AFLplusplus-4.00c/GNUmakefile.llvm
@@ -50,7 +50,7 @@
 LLVM_NEW_API = $(shell $(LLVM_CONFIG) --version 2>/dev/null | egrep -q '^1[0-9]' && echo 1 || echo 0 )
 LLVM_10_OK = $(shell $(LLVM_CONFIG) --version 2>/dev/null | egrep -q '^1[1-9]|^10\.[1-9]|^10\.0.[1-9]' && echo 1 || echo 0 )
 LLVM_HAVE_LTO = $(shell $(LLVM_CONFIG) --version 2>/dev/null | egrep -q '^1[1-9]' && echo 1 || echo 0 )
-LLVM_BINDIR = $(shell $(LLVM_CONFIG) --bindir 2>/dev/null)
+LLVM_BINDIR = /usr/bin
 LLVM_LIBDIR = $(shell $(LLVM_CONFIG) --libdir 2>/dev/null)
 LLVM_STDCXX = gnu++11
 LLVM_APPLE_XCODE = $(shell $(CC) -v 2>&1 | grep -q Apple && echo 1 || echo 0)
